index.js

```diff
@@ -118,10 +133,13 @@ function getLastCarInfo(/* code here */) {
  * For example, if getCarInfoById is invoked with the inventory and the number 1,
  * it will return `This is a Lincoln Navigator`.
 */
-function getCarInfoById(/* code here */) {
-  /* code here */
+function getCarInfoById(array,id) {
+
+    const getCar = array.find(array => array.id === id);
+    return `This is a  ${getCar.car_make}  ${getCar.car_model}`;
 }
 
```

Comment:
You were not supposed to use the .find method on this project. Instead you should have used an if statement inside a for loop.